documentation/projects/proposals/trust_and_safety/content_report_moderation/20230517-implementation_plan_mature_copy_change.md

```diff
@@ -0,0 +1,286 @@
+- [Search store references](https://github.com/WordPress/openverse/blob/92ea64c6d4ac1801785eadf99a3782628eadad06/frontend/src/stores/search.ts)
+- [Content report utilities](https://github.com/WordPress/openverse/blob/74da3d852a45c514c0159038c86164a26d70504b/frontend/src/utils/content-safety.ts#L10-L23)
+- All affected tests.
+
+The following references _will not be_ affected because they correspond to
+either the data model or to the API query parameter `mature`. The query
+parameter will be changed to `include_sensitive_results` in the API
+[as part of another project](https://docs.openverse.org/projects/proposals/trust_and_safety/detecting_sensitive_textual_content/20230309-project_proposal_detecting_sensitive_textual_content.html#filtering)
+
+- [Search query transform utilities](https://github.com/WordPress/openverse/blob/92ea64c6d4ac1801785eadf99a3782628eadad06/frontend/src/utils/search-query-transform.ts#L31)
+  _(query parameter)_
+- [`Media` interface attributes](https://github.com/WordPress/openverse/blob/930a2ec4583d53e2f17e7aa397dc9b3b32cc8ebb/frontend/src/types/media.ts#L48)
+  _(data model)_
+
+#### Copy
+
+Almost all copy changes necessary can be made in the
+[`en.json5` base translations file](https://github.com/WordPress/openverse/blob/2041c5df1e9d5d1f9f37e7c177f2e70f61ea5dba/frontend/src/locales/scripts/en.json5).
+The `filters.mature` key can be removed entirely, as it references a filter
+which no longer exists.
+
+## Dependencies
+
+### Infrastructure
+
+<!-- Describe any infrastructure that will need to be provisioned or modified. In particular, identify associated potential cost changes. -->
+
+This work will include several migrations for the API, which will need to be
+deployed one after another in production as described in
+[the API section](#api).
+
+### Tools & packages
+
+<!-- Describe any tools or packages which this work might be dependent on. If multiple options are available, try to list as many as are reasonable with your own recommendation. -->
+
+No other tools or packages should be required.
+
+### Other projects or work
+
+<!-- Note any projects this plan is dependent on. -->
+
+This work intersects with, but is distinct from, the
+["Detecting and Blurring Sensitive Textual Content" project](https://docs.openverse.org/projects/proposals/trust_and_safety/detecting_sensitive_textual_content/index.html).
+
+## Design
+
+<!-- Note any design requirements for this plan. -->
+
+No design work should be necessary.
+
+## Parallelizable streams
+
+<!-- What, if any, work within this plan can be parallelized? -->
+
+The frontend copy work and the API copy & code work can be done simultaneously.
+The frontend code work should wait until the API work is complete so the content
+reporting using `sensitive` rather than `mature` is available on the API prior
+to the frontend code changes.
```

Comment:
This probably needs to take into account the changes that will happen in #2118, primarily to note that we'll need to coordinate around that.

I don't think the frontend copy changes need to be blocked on the API work though, which seems to be what the first sentence in this section says.

Not a blocker, just clarification. I think this coordination will happen naturally as the work on the frontend copy gets done. Maybe just adding a note to the frontend issues to be aware that the other project will also be making related changes and to be aware of potential overlaps/merge conflicts that might arise.